daemon/list.go

```diff
@@ -126,15 +135,15 @@ func (daemon *Daemon) Containers(job *engine.Job) error {
 			return nil
 		}
 		displayed++
-		out := &engine.Env{}
-		out.SetJson("Id", container.ID)
-		out.SetList("Names", names[container.ID])
+		newC := *new(types.Container)
```

Comment:
I don't understand this trick. Is it just `types.Container{ID: container.ID, Names: names[container.ID]}` ?